en/gps_compass/septentrio_asterx-rib.md

```diff
@@ -0,0 +1,209 @@
+is required to set the layout as accurately as possible in the web app.
+:::
+
+### Dual antenna
+
+The attitude (heading/pitch) can be computed from the orientation of the baseline between the main and the aux1 GNSS
+antennas.
+
+<div style="text-align: center;">
+
+![Multi-antenna attitude determination setup.]( ../../assets/hardware/gps/septentrio_sbf/multi-antenna_attitude_setup.png "Multi-antenna attitude determination setup.")
+
+</div>
+
+To enable multi-antenna attitude determination, follow the following procedure:
+
+1. Attach two antennas to your vehicle, using cables of approximately the same length. The default antenna configuration
+   is as depicted in the figure. It consists in placing the antennas aligned with the longitudinal axis of the vehicle,
+   main antenna behind aux1. For best accuracy, try to maximize the distance between the antennas, and avoid significant
+   height difference between the antenna ARPs.
+2. In practice, the two antenna ARPs may not be exactly at the same height in the vehicle
+   frame, or the main-aux1 baseline may not be exactly parallel or perpendicular to the
+   longitudinal axis of the vehicle. This leads to offsets in the computed attitude angles.
+   These offsets can be compensated for with the **setAttitudeOffset** command.
+
+:::note 
+For optimal heading results, the two antennas should be seperated by at least 30cm / 11.8 in (ideally 50cm /
+19.7in or more)
+
+For additional configuration of the dual antenna setup, please refer to
+our [Knowledge Base](https://customersupport.septentrio.com/s/article/Setting-up-and-configuring-the-AsteRx-i-compensating-for-a-non-default-antenna-orientation) or the [hardware manual](https://www.septentrio.com/system/files/support/asterx-i3_product_group_hardware_manual_1.0.0.pdf)
+:::
+
+### Web app
+
+mosaic-H GPS/GNSS receiver module with heading comes with fully documented interfaces, commands and data messages. The
+included GNSS receiver control and analysis software [RxTools](https://www.septentrio.com/en/products/software/rxtools)
+allows receiver configuration, monitoring as well as data logging and analysis.
+
+The receiver includes an intuitive web user interface for easy operation and monitoring allowing you to control the
+receiver from any mobile device or computer. The web interface also uses easy-to-read quality indicators ideal to
+monitor the receiver operation during the job at hand.
+
+<div style="text-align: center;">
+
+![Illustrative image for Septentrio mosaic-H GNSS Receiver Module Web User Interface (WebUI)]( ../../assets/hardware/gps/septentrio_sbf/Septentrio-mosaic-X5-H-T-CLAS-GNSS-Module-Receiver-WebUI.png "Septentrio web user interface")
+
+</div>
+
+## PX4 setup
+
+<div style="text-align: center;">
+
+![QGroundControl parameter settings](../../assets/hardware/gps/septentrio_sbf/QGC_param.png "QGroundControl parameter settings")
+
+</div>
+
+### Single antenna
+
+Edit the following parameters in the GPS tab:

```

Comment:
`TELEM1` is generally used for the GCS telemetry stream, so doing this will stop the GCS connecting. Any reason you chose not to use GPS port for GPS? https://docs.px4.io/master/en/peripherals/mavlink_peripherals.html#telem1

Reply:
I have always used the `TELEM1` port because I only have a 6pin cable available. It is indeed better to use the `GPS` port, this should not be a problem. I will update the schematics and configuration.